lib/correios/sro/xml_builder.rb

```diff
@@ -0,0 +1,58 @@
+module Correios
+  module SRO
+    class XMLBuilder
```

Comment:
How about a better name that reflects the responsibility of this class? Maybe changing `build` method for something like `build_request` or `build_request_body` is enough. 

What do you think?